app/views/traces/edit.html.erb

```diff
@@ -4,52 +4,15 @@
 
 <img src="<%= url_for :controller => "traces", :action => "picture", :id => @trace.id, :display_name => @trace.user.display_name %>">
 
-<%= form_for @trace do |f| %>
-
-<div id='edit-trace-form' class='standard-form'>
-  <fieldset>
-    <div class='standard-form-row'>
-      <label class='standard-label'><%= t ".filename" %></label>
-      <p class='deemphasize'><%= @trace.name %> (<%= link_to t(".download"), trace_data_path(@trace) %>)</p>
-    </div>
-    <div class='standard-form-row'>
-      <label class='standard-label'><%= t ".uploaded_at" %></label>
-      <p class='deemphasize'><%= l @trace.timestamp, :format => :friendly %></p>
-    </div>
-  <% if @trace.inserted? %>
-    <div class='standard-form-row'>
-      <label class='standard-label'><%= t ".points" %></label>
-      <p class='deemphasize'><%= @trace.size.to_s.gsub(/(\d)(?=(\d{3})+$)/, '\1,') %></p>
-    </div>
-    <div class='standard-form-row'>
-      <label class='standard-label'><%= t ".start_coord" %></label>
-    </div>
-    <div class="geo">
-      <span class="latitude"><%= @trace.latitude %></span>;
-      <span class="longitude"><%= @trace.longitude %></span>
-    </div>
-    (<%= link_to t(".map"), :controller => "site", :action => "index", :anchor => "map=14/#{@trace.latitude}/#{@trace.longitude}" %> / <%= link_to t(".edit"), :controller => "site", :action => "edit", :gpx => @trace.id, :anchor => "map=14/#{@trace.latitude}/#{@trace.longitude}" %>)
-  <% end %>
-    <div class='standard-form-row'>
-      <label class='standard-label'><%= t ".owner" %></label>
-      <p class='deemphasize'><%= link_to @trace.user.display_name, user_path(@trace.user) %></p>
-    </div>
-    <div class='standard-form-row'>
-      <label class='standard-label'><%= t ".description" %></label>
-      <%= f.text_field :description %>
-    </div>
-    <div class='standard-form-row'>
-      <label class='standard-label'><%= t ".tags" %></label>
-      <%= f.text_field :tagstring %> (<%= t ".tags_help" %>)
-    </div>
-    <div class='standard-form-row'>
-      <label class='standard-label'><%= t ".visibility" %></label>
-      <%= f.select :visibility, [[t("traces.visibility.private"), "private"], [t("traces.visibility.public"), "public"], [t("traces.visibility.trackable"), "trackable"], [t("traces.visibility.identifiable"), "identifiable"]] %> (<a href="<%= t ".visibility_help_url" %>"><%= t ".visibility_help" %></a>)
-    </div>
-  </fieldset>
-
-</div>
-
-<%= f.submit %>
-
+<%= simple_form_for @trace do |f| %>
```

Comment:
This seems to have lost all the extra information that was displayed before the editable fields - is that a deliberate decision?

Reply:
Yeah, no final decision on that. I'm not sure why e.g. number of points was shown on the edit form, it's not like you could change any of that metadata so why have it shown disabled on the form. Feedback welcome on this though.

I started playing around with the simple_form `disabled` option and the bootstrap `.form-control-plaintext` class but haven't finished that line of enquiry.